studio/pages/account/me.tsx

```diff
@@ -22,7 +22,7 @@ const User: NextPageWithLayout = () => {
 
 User.getLayout = (page) => (
   <AccountLayout
-    title="Supabase"
+    title="Preferences"
```

Comment:
@joshenlim I updated these page titles, IMO it makes more sense than just "Supabase | Supabase" like current version. WDYT?

Reply:
cc @saltcod